johnzon-jsonb/src/test/java/org/apache/johnzon/jsonb/JsonbRecordTest.java

```diff
@@ -34,19 +34,19 @@ public class JsonbRecordTest {
 
     @Test
     public void roundTrip() {
-        final Record ref = new Record(119, "Santa");
+        final JsonbRecord ref = new JsonbRecord(119, "Santa");
         final String expectedJson = "{\"_name\":\"Santa\",\"age\":119}";
         assertEquals(expectedJson, jsonb.toJson(ref));
-        assertEquals(ref, jsonb.fromJson(expectedJson, Record.class));
+        assertEquals(ref, jsonb.fromJson(expectedJson, JsonbRecord.class));
     }
 
     @JohnzonRecord
-    public static class Record {
+    public static class JsonbRecord {
```

Comment:
why was it needed?

Reply:
yes, was the intent - and to be honest it is not lowercase just to run well on later java versions ;) - cause it is supposed to mimic a `record`